rasterio/_shim30.pyx

```diff
@@ -90,3 +91,12 @@ cdef const char* osr_get_name(OGRSpatialReferenceH hSrs):
 
 cdef void osr_set_traditional_axis_mapping_strategy(OGRSpatialReferenceH hSrs):
     OSRSetAxisMappingStrategy(hSrs, OAMS_TRADITIONAL_GIS_ORDER)
+
+
+cdef void set_proj_search_path(object path):
+    cdef const char *const *paths = NULL
```

Comment:
Just a reminder for https://github.com/Toblerity/Fiona/pull/826

Reply:
Yep!